examples/collection-get-item/index.tsx

```diff
@@ -0,0 +1,26 @@
+import "./style.css";
+import * as Ariakit from "@ariakit/react";
+
+type CustomItem = {
+  id: string;
+  element?: HTMLElement | null | undefined;
+  color?: string;
+};
+
+export default function Example() {
+  const collection = Ariakit.useCollectionStore<CustomItem>({ items: [] });
```

Comment:
There's a overload in `useCollectionStore` that forces me pass items or defaultItems to props if I pass a custom Item type. Not sure why it's needed

Reply:
> There's a overload in `useCollectionStore` that forces me pass `items` or `defaultItems` props if I pass a custom `Item` type. Not sure why it's needed

Yes, it should not be needed. I'm not sure how trivial it is to change that, though.

Anyway, since the items are *uncontrolled* here, it should be `defaultItems`:
```suggestion
  const collection = Ariakit.useCollectionStore<CustomItem>({
    defaultItems: [],
  });
```